core-chart/values.yaml

```diff
@@ -0,0 +1,61 @@
+# Default values for KubeStellar-Core.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+
+# Container/chart image versions. These values are set during the chart release process,
+# please do not change them unless you know what your are doing.
+HELM_VERSION: "3.15.0"
+KUBECTL_VERSION: "1.30.1"
+KUBESTELLAR_VERSION: "0.23.0-alpha.2"
+CLUSTERADM_VERSION: "0.8.2"
+OCM_STATUS_ADDON_VERSION: "0.2.0-rc8"
+OCM_TRANSPORT_PLUGIN_VERSION: "0.1.8"
+
+
+# Control controller log verbosity
+# The "default" verbosity value will be used for all controllers unless a specific controller verbosity override is specified
+verbosity:
+  default: 2
+  # Specific controller verbosity overrides:
+  # kubestellar: 6
+  # clusteradm: 6
+  # status_controller: 6
+  # status_agent: 6
+  # transport: 6
+
```

Comment:
FYI, also coming into existence: controllers taking command line flags for request QPS and burst.